modules/core/src/main/java/org/apache/ignite/internal/jdbc/thin/JdbcThinConnection.java

```diff
@@ -1122,6 +1140,8 @@ JdbcResultWithIo sendRequest(JdbcRequest req, JdbcThinStatement stmt, @Nullable
                     if (req instanceof JdbcQueryExecuteRequest)
                         qryReq = (JdbcQueryExecuteRequest)req;
 
+                    req.clientInfo(clientInfo);
```

Comment:
Why do we send client info with every request, but read it only for `JdbcQueryExecuteRequest` and `JdbcBatchExecuteRequest`?